rotkehlchen/tests/api/test_history_export.py

```diff
@@ -0,0 +1,152 @@
+import csv
```

Comment:
please rename the file to: `test_history_events_export.py`


Test history export can also be confused to be the PnL history as that name is used a lot there